controller/api/destination/profile_translator.go

```diff
@@ -41,10 +49,11 @@ func (pt *profileTranslator) Update(profile *sp.ServiceProfile) {
 	pt.stream.Send(destinationProfile)
 }
 
-func defaultServiceProfile() *pb.DestinationProfile {
+func defaultServiceProfile(fullyQualifiedName string) *pb.DestinationProfile {
```

Comment:
TIOLI:  I think this would make sense to be a method on `profileTranslator` which would mean we wouldn't need to pass in the FQN anymore.